tests/benchmark/test_route_resolution_benchmark.py

```diff
@@ -13,6 +16,34 @@
 sanic.router.ROUTER_CACHE_SIZE = 0
 
 
+# noinspection PyBroadException
+class TestURLParseBenchmark:
```

Comment:
I think we can keep this out. I do not think there is a need to keep this in the core repo right now. If you want to keep it somewhere safe for later usage, I would suggest a gist, or open an new repo under `sanic-org`.